src/v2/Apps/Order/Routes/__tests__/Offer.jest.tsx

```diff
@@ -202,6 +207,10 @@ describe("Offer InitialMutation", () => {
       expect(page.offerInput.text()).toMatch("Offer amount missing or invalid.")
     })
 
+    it("adds a custom note given no note present", async () => {
```

Comment:
when retiring the feature flag we must add the test case here, right now is too complicated with this test file to mock some stuff only in a single test case

Reply:
Impact Over Perfection 🚀